.env.example

```diff
@@ -3,3 +3,5 @@ POSTGRES_PASSWORD=password
 POSTGRES_DB=database
 POSTGRES_PORT=port
 POSTGRES_SERVER=server
+SECRET_KEY=secret_key
+ALGORITHM=algorithm
```

Comment:
please name these vars so they represent what they are used for. Additionally, I don't think you should allow change of algorithm here. Otherwise old records in db with passwords won't work